server/utils/db/db_utils.py

```
@@ -2986,6 +2999,20 @@ def initialize_tables():
                 conn.rollback()
 
+            # Migration: Add onboarding_completed column to organizations table
+            try:
+                cursor.execute(
+                    "ALTER TABLE organizations ADD COLUMN IF NOT EXISTS onboarding_completed BOOLEAN DEFAULT FALSE;"
+                )
+                conn.commit()
+                logging.info("Ensured onboarding_completed column exists on organizations table.")
+            except Exception as e:
+                logging.exception(
+                    "FATAL: Failed to ensure organizations.onboarding_completed: %s", e
```

Ruff (0.15.18)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)